package.json

```diff
@@ -45,32 +45,32 @@
     "react": "^15.3.0 || ^16.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.7.4",
-    "@babel/core": "^7.7.4",
-    "@babel/preset-typescript": "^7.7.4",
-    "@researchgate/babel-preset": "^1.5.0",
-    "@researchgate/spire-config": "^2.0.0",
-    "@types/classnames": "^2.2.9",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.9.6",
+    "@babel/preset-typescript": "^7.9.0",
+    "@researchgate/babel-preset": "^1.5.2",
+    "@researchgate/spire-config": "^2.1.5",
+    "@types/classnames": "^2.2.10",
     "@types/prop-types": "^15.7.3",
-    "@types/react": "^16.9.13",
-    "cross-env": "^6.0.3",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.15.1",
-    "enzyme-to-json": "^3.4.3",
-    "jest-snapshot": "^24.9.0",
-    "npm-run-all": "^4.1.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.12.0",
-    "rimraf": "^3.0.0",
+    "@types/react": "^16.9.34",
+    "cross-env": "^7.0.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.4.4",
+    "jest-snapshot": "^25.5.1",
+    "npm-run-all": "^4.1.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "rimraf": "^3.0.2",
     "spire": "^1.8.3",
     "spire-plugin-semantic-release": "^1.8.2",
-    "typescript": "^3.7.2"
+    "typescript": "^3.8.3"
   },
   "dependencies": {
     "@types/highlight.js": "^9.12.3",
-    "classnames": "^2.2.3",
-    "highlight.js": "^9.0.0",
-    "prop-types": "^15.5.6"
+    "classnames": "^2.2.6",
+    "highlight.js": "^10.0.1",
+    "prop-types": "^15.7.2"
```

Comment:
We shouldn't update all dependencies in package.json unless we know that one of them is the source of a security vul. Having bigger ranges here helps npm/yarn to deduplicate versions easily.